src/types/entity/File.d.ts

```diff
@@ -0,0 +1,14 @@
+import Entity from './Entity';
+
+declare class File extends Entity {
```

Comment:
in general with each declaration it would be good to group by method, separating by newline symbol, like:

```
setContent()
getContent()

setMimeType()
getMimeType()
```

this would improve readability

Reply:
sure, it will be more clear, thanks